src/NoDelegateCall.sol

```diff
@@ -1,9 +1,13 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.20;
 
+import {CustomRevert} from "./libraries/CustomRevert.sol";
```

Comment:
could do `using CustomRevert for bytes4 global`?
I actually think we should do that with more of our libs too

Reply:
Unfortunately the `global` keyword only works for user-defined types.
https://docs.soliditylang.org/en/latest/contracts.html#using-for